### PR TITLE
Feature/new sui react initial props

### PR DESCRIPTION
**Pull request by miduga**
Create new package @schibstedspain/sui-react-initial-props
> Make your React pages to get initial props asynchronously both client and server

## Motivation

**sui-react-initial-props** offers a way to make your easily your app isomorphic.
* Offers same parameters for your getInitialProps in the client in the server to make your app 100% universal.
* Avoid re-renders as other options like React-Transmit causing a longer time to respond, specially in the server.
* Minimal footprint by focusing on the really need stuf.

### CODE REVIEW

File: packages/sui-react-initial-props/src/createClientPageComponent.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+// This method creates the PageComponent prepared for the Client Side Rendering
+// It renders a placeholder, if any specified as `renderLoading` on the PageComponent
+// Also, executes the getInitialProps to retrieve the initialProps and then
+// Renders the component with the initialProps, hiding the placeholder
+export default ({ Page, context, routeInfo }) => (
```
Comment (carlosvillu) [inline packages/sui-react-initial-props/src/createClientPageComponent.js]: You rewrite this HoC in the way:

```
export default ({context, routeInfo}) => Page => class ClientPage extends Component {
  static displayName = `ClientPage(${Page.displayName})`
}
```

Doing that we can combine this HoC with others

File: packages/sui-react-initial-props/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "lib": "rm -Rf ./lib && mkdir -p ./lib && babel --presets sui ./src --out-dir ./lib",
+    "prepublish": "npm run lib"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "peerDependencies": {
+    "react": "15"
+  },
+  "devDependencies": {
```
Comment (miduga) [inline packages/sui-react-initial-props/package.json]: Yes !!?!?!?! 💃

File: packages/sui-react-initial-props/src/createClientPageComponent.js (hunk 1 above)
Comment (miduga) [inline packages/sui-react-initial-props/src/createClientPageComponent.js]: 👍

File: packages/sui-react-initial-props/package.json (hunk 2 above)
Comment (miduga) [inline packages/sui-react-initial-props/package.json]: Done :)

**Comment by miduga** [comment]
@carlosvillu Comments addressed. ;) Take a look to it, please!

**Pull request merged**